Library/Homebrew/extend/ARGV.rb

```diff
@@ -120,6 +120,10 @@ def dry_run?
     include?("--dry-run") || switch?("n")
   end
 
+  def keep_files?
+    flag? "--keep-files"
+  end
```

Comment:
Another thought is if we only use it in `test`, we probably want to avoid adding this method to `ARGV`. Just use `ARGV.include? "--keep-files"` directly in `test.rb`